applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py

```diff
@@ -284,81 +311,76 @@ def _add_variables_to_model_part(self, model_part):
     def _model_part_reading(self):
         # reading the model part of the background grid
         if(self.settings["grid_model_import_settings"]["input_type"].GetString() == "mdpa"):
-            KratosMultiphysics.ModelPartIO(self.settings["grid_model_import_settings"]["input_filename"].GetString()).ReadModelPart(self.model_part1)
+            KratosMultiphysics.ModelPartIO(self.settings["grid_model_import_settings"]["input_filename"].GetString()).ReadModelPart(self.grid_model_part)
```

Comment:
please don't do this manually, there is a function for importing modelparts in the `PythonSolver`